ctapipe/tools/display_dl1.py

```diff
@@ -136,22 +136,19 @@ class DisplayDL1Calib(Tool):
         help="Telescope to view. Set to None to display all telescopes.",
     ).tag(config=True)
 
-    aliases = Dict(
-        dict(
-            input="EventSource.input_url",
-            max_events="EventSource.max_events",
-            T="DisplayDL1Calib.telescope",
-            O="ImagePlotter.output_path",
+    aliases = {
+        ("i", "input"): "EventSource.input_url",
+        ("m", "max-events"): "EventSource.max_events",
+        "T": "DisplayDL1Calib.telescope",
```

Comment:
```suggestion
        ("t", "telescope"): "DisplayDL1Calib.telescope",
```

Reply:
Ahh yes, I wanted to ask if this would be a good change